qemu/tests/balloon_minimum.py

```diff
@@ -41,5 +41,7 @@ def run(test, params, env):
         time.sleep(30)
         repeat_times -= 1
     ballooned_memory = balloon_test.ori_mem - expect_mem
+    balloon_test.pre_mem = balloon_test.get_ballooned_memory()
+    balloon_test.pre_gmem = balloon_test.get_memory_status()
```

Comment:
Should move to before do balloon, aka. L34?

Reply:
Update